test/fake-timers-test.js

```diff
@@ -4504,19 +4504,6 @@ describe("FakeTimers", function() {
             assert(called);
             clock.uninstall();
         });
-
-        it("does not install by default - GitHub#126", function(done) {
```

Comment:
Would it be possible to change this test to assert nextTick is mocked if the behavior changes?

Reply:
changed the test above, I think that's fine?

Reply:
> changed the test above, I think that's fine?

Yes, this is fine. It's implicitly true, as the code won't work unless `nextTick` is synchronous.